includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -2097,6 +2097,17 @@ public function add_order_actions( $actions ) {
 			return $actions;
 		}
 
+		// if order is already completed, we shouldn't capture the charge anymore.
+		if (
+			in_array(
+				$theorder->get_status(),
+				[ WC_Payments_Order_Service::STATUS_PROCESSING, WC_Payments_Order_Service::STATUS_COMPLETED ],
```

Comment:
The hard coded list is not friendly to 3rd party extensions - the statuses can be overridden by `woocommerce_order_is_paid_statuses` hook. Consider using `wc_get_is_paid_statuses` function instead.

Reply:
fixed in [8f955ce](https://github.com/Automattic/woocommerce-payments/pull/5285/commits/8f955cea51219f79877bffd9b553491bc7b2a940)